sycl/source/detail/device_info.hpp

```diff
@@ -983,7 +983,8 @@ struct get_device_info_impl<
     // and LevelZero.
     return (Dev->getBackend() == backend::ext_oneapi_level_zero) ||
            (Dev->getBackend() == backend::opencl) ||
-           (Dev->getBackend() == backend::ext_oneapi_cuda);
+           (Dev->getBackend() == backend::ext_oneapi_cuda) ||
+           (Dev->getBackend() == backend::ext_oneapi_hip);
```

Comment:
The comment above needs to be changed:

// Currently fusion is only supported for SPIR-V based backends, i.e. OpenCL

Reply:
Thanks for the review, I fixed the comment with [eaed64e](https://github.com/intel/llvm/pull/11003/commits/eaed64e356b5f16ecadc1bde7814504ab70858d0)